core/src/lib.rs

```diff
@@ -30,6 +32,8 @@ pub use crate::model::errors::*;
 pub use crate::service::import_export_service::{ImportExportFileInfo, ImportStatus};
 pub use crate::service::sync_service::{SyncProgress, WorkCalculated};
 pub use crate::service::usage_service::{bytes_to_human, UsageItemMetric, UsageMetrics};
+pub use base64;
+pub use libsecp256k1::PublicKey;
```

Comment:
Please toss these with the other third party pub uses (uuid above) and not with the crates.